### ISSUE TITLE
Predbat does not call discharge_stop_service

### ISSUE THREAD

**Issue by jacekowski** [springfall2008/batpred#926]
I've been working on integrating predbat with my sunsynk inverter and i've found what i think might be a bug

In this line charge_stop_service is being called instead of discharge_stop_service, is this intentional for some reason (so charge_stop_service is supposed to stop both discharge and charge) or is it a typo?
https://github.com/springfall2008/batpred/blob/3b5ffa6fa838cc95e47b360cfe01aad2a575a497/apps/predbat/predbat.py#L3759

**Comment by springfall2008** [comment]
This is actually by design in that the stop is the same for charge or discharge, is it not in your case?

**Comment by jacekowski** [comment]
Not with my inverter. I'm got a sunsynk inverter, so i'm running some node red between it and predbat to make things actually happen (i've actually started with pv opt which led to some design choices).

Alternatively ability to call service with multiple entity id's would also work (with example below only first entity will be passed to actual service call):

```
  charge_stop_service:
      service: switch.turn_off
      entity_id:.
        - "switch.schedule_charge"
        - "switch.schedule_discharge"
```

**Comment by springfall2008** [comment]
> Not with my inverter. I'm got a sunsynk inverter, so i'm running some node red between it and predbat to make things actually happen (i've actually started with pv opt which led to some design choices, but i also need some other workarounds for inverter quirks).
> 
> Alternatively ability to call service with multiple entity id's would also work (with example below only first entity will be passed to actual service call):
> 
> ```
>   charge_stop_service:
>       service: switch.turn_off
>       entity_id:.
>         - "switch.schedule_charge"
>         - "switch.schedule_discharge"
> ```

This is certainly doable add I could add it.

That said, you can just make an automation in HA which does this and then have Predbat trigger the automation as the service.

It's also worth pointing out that Predbat can control the scheduled charge and discharge switch directly if you configure them in apps.yaml and enable it in the inverter configuration

**Comment by springfall2008** [comment]
That said scheduled charge means to the time window configured, so you will need to make sure the time window is set from Predbat too.

**Comment by jacekowski** [comment]
I've realised that after spending some time diagnosing why things are not working as expected. 

I have also disabled service api for now and things seem to work.

It would be nice if predbat could provide time and date (using time & date integration) instead of just time for charge/discharge window.

On my inverter (sunsynk) there is no way to schedule charge/discharge (it has scheduling, but it is limited to what can be done as setting needed to discharge/charge is a global setting not one that is changed with a schedule) so i have node-red adjusting settings as needed when needed.
